endoreg_db/views/finding_classification/base.py

```
@@ -0,0 +1,45 @@
+# endoreg_db/views/classification_views.py
+from rest_framework.viewsets import ReadOnlyModelViewSet
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from endoreg_db.models import (
+    FindingClassification,
+)
+from ...serializers.finding import (
+    FindingClassificationSerializer,
+)
```

Pylint (3.3.7)

[error] No name 'FindingClassificationSerializer' in module 'endoreg_db.serializers.finding'

(E0611)